src/newrelic.coffee

```diff
@@ -21,7 +21,8 @@
 #   hubot newrelic servers - Returns statistics for all servers from New Relic
 #   hubot newrelic servers name <filter_string> - Returns a filtered list of servers
 #   hubot newrelic users - Returns a list of all account users from New Relic
-#   hubot newrelic user email <filter_string> - Returns a filtered list of account users
+#   hubot newrelic users email <filter_string> - Returns a filtered list of account users
+#   hubot newrelic user emails - Returns a list of all user emails
```

Comment:
I'm curious why you decided to make `user` singular here. Wouldn't it be more intuitive to keep them all `users`?